### ISSUE TITLE
HTML support?

### ISSUE THREAD

**Issue by sashabeep** [sachinchoolur/lightGallery#1136]
## Summary

Possibility to display any HTML content

### Basic example

It would be great to have ability to display any html content (from hidden layer, or ajax, whatever) to use single lightbox for all the modals on the website.

### Motivation

It will be easier to maintain single plugin for all the popup/modals, etc

**Comment by pixedelic** [comment]
👍  it would be great

**Comment by ChipRyan** [comment]
+1

**Comment by MangoMarcus** [comment]
+1

**Comment by niktor76** [comment]
+1

**Comment by janklimo** [comment]
Agreed, this would let me migrate everything to lightGallery. What I'm looking for is having the ability to have basic image slides mixed with slides with HTML content like this:

<img width="1791" alt="Screenshot 2023-09-01 at 16 44 46" src="https://github.com/sachinchoolur/lightGallery/assets/7811733/a6c93d3d-bafc-4bdd-b311-6aa0734990b6">
